alerts_consumer.go

```diff
@@ -275,42 +243,23 @@ func (c *AlertsConsumer) SendBatch(batch [][]byte, tag string) error {
 			return err
 		}
 
-		pts = append(pts, eo.Datapoints...)
-		evts = append(evts, eo.Events...)
-		dats = append(dats, eo.MetricData...)
+		metrics = append(metrics, eo.DDMetrics...)
+		dats = append(dats, eo.CWMetrics...)
 	}
 
-	for idx := range pts {
-		updateMaxDelay(pts[idx].Timestamp)
-
-		// For fairly recent logs, let SignalFX assign a timestamp on arrival,
-		// instead of the log's actual timestamp.  This ensures datapoints are sent
-		// in-order per timeseries. (If out of order, SignalFX will drop them
-		// silently.)
-		//
-		// For older logs, use their actual timestamp. This may result in some
-		// dropped logs, but should be less important since we care mainly about
-		// accurate alerting on recent data.
-		if isRecent(pts[idx].Timestamp, 30*time.Second) {
```

Comment:
I think this is no longer an issue for DD, but someone can correct me if wrong.

Reply:
yeah i think this is ok